cypress/integration/user/certifications.js

```diff
@@ -0,0 +1,52 @@
+describe('Public profile certifications', () => {
+  context('Signed in user viewing their own public profile', () => {
+    before(() => {
+      cy.exec('npm run seed:certified-user');
+    });
+
+    beforeEach(() => {
+      cy.login();
+    });
+
+    it('Should show claimed certifications if the username has all lowercase characters', () => {
+      cy.visit('/certifieduser', { failOnStatusCode: false });
+
+      // The following line is only required if you want to test it in development
+      // cy.contains('Preview custom 404 page').click();
+
+      cy.get('.certifications.row > div')
+        .children()
+        .should('contain', 'freeCodeCamp Certifications')
+        .and('contain', 'Legacy Certifications')
+        .get('.certifications > .btn')
```

Comment:
Traversing the DOM like this is something of an anti-pattern for Cypress.  All this test is really trying to do is answer the question "are there certification links in the page", but it will fail if we reorganise those buttons or use different classes to style them.

A more robust approach would be to add data-cy attributes to the buttons and assert that we see the expected number (since we know how many certifieduser has claimed).

Finally, a couple of snapshot tests to lock down the UI would be awesome.  I'd be happy to add those in, if that sounds reasonable.